airbyte-integrations/connectors/source-firebolt/setup.py

```diff
@@ -5,7 +5,7 @@
 
 from setuptools import find_packages, setup
 
-MAIN_REQUIREMENTS = ["airbyte-cdk", "firebolt-sdk>=0.8.0"]
+MAIN_REQUIREMENTS = ["airbyte-cdk~=0.2", "firebolt-sdk>=0.8.0"]
```

Comment:
https://github.com/firebolt-db/firebolt-python-sdk/blob/0.11.0/setup.cfg#L32

It doesn't seem work. I run 
```
pip install -r requirements.txt
```

in my local environment, I got the same error as CI. 

```
The conflict is caused by:
    source-acceptance-test 0.0.0 depends on pydantic~=1.6
    airbyte-cdk 0.2.0 depends on pydantic~=1.9.2
    pydantic[dotenv] 1.8.2 depends on pydantic 1.8.2 (from https://files.pythonhosted.org/packages/ff/74/54e030641601112309f6d2af620774e9080f99c7a15742fc6a0b170c4076/pydantic-1.8.2-py3-none-any.whl#sha256=fec866a0b59f372b7e776f2d7308511784dace622e0992a0b59ea3ccee0ae833 (from https://pypi.org/simple/pydantic/) (requires-python:>=3.6.1))

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

```

Firebolt uses `pydantic[dotenv]==1.8.2`, which is unchanged for a while. @sajarin Can you please advise?

Reply:
I believe this was caused by a change that bumped airbyte-cdk from 0.1.80 to 0.1.81 - https://github.com/airbytehq/airbyte/commit/3cfa4892342fba109e7670b4c51b354282fdadc0

Since the original version of Firebolt Source was merged before that change it worked with no issues.

One solution for this would be to pin cdk version to <=0.1.80 in firebolt-source for this change.

Reply:
`source-acceptance-test 0.0.0 depends on airbyte-cdk~=0.2`, so the minimal version of airbyte-cdk is 0.2

Reply:
Yeah, you're right. `source-acceptance-test` does not have versions so it forces us to use airbyte-cdk>=0.2. Let me see if I can relax the requirement on `firebolt-sdk`.

Reply:
@YiyangLi I've relaxed the requirements on firebolt-sdk==0.12.0 so the tests should be succeeding now. Could you re-run the workflow?